packages/forma-36-react-components/.eslintrc.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   rules: {
     'react/prop-types': 'off',
     'react/prefer-stateless-function': [0, { ignorePureComponents: true }],
-    'react/jsx-filename-extension': { extensions: ['.js', '.jsx'] },
+    'react/jsx-filename-extension': [0, { extensions: ['.js', '.jsx'] }],
```

Comment:
Question: Why the `[0]` thingy?

Reply:
To make it an error. The old setting actually didn't work because it was missing the array and error level. I'll change it (and the other 0 values) to a string to make it more obvious.